samples/batch/csharp/batchclient/dto/dtobase.cs

```diff
@@ -8,30 +8,20 @@ namespace BatchClient
     using System;
     using System.Collections.Generic;
 
-    public sealed class Transcription
+    public abstract class DtoBase
```

Comment:
The sample does not use any database object, can we call it another name?

Reply:
Dto == data transfer object. Usuallz the thing that gets transferred over the wire. I think this is a pretty good name and has nothing to do with db.